### PR TITLE
Qt,BaseGL,BrushingAndLinking,DataFrameQt,Volume: warn fixes

### PR DESCRIPTION
**Pull request by martinfalk**

### CODE REVIEW

File: modules/fontrendering/src/processors/textoverlaygl.cpp [hunk 1]
```diff
@@ -46,6 +46,7 @@
 #include <locale>
 
 #include <fmt/format.h>
+#include <fmt/args.h>
```
Comment (martinfalk) [inline modules/fontrendering/src/processors/textoverlaygl.cpp:51]: <fmt/args.h> no longer exists
https://fmt.dev/latest/api.html

File: cmake/compileoptions.cmake [hunk 2]
```diff
@@ -49,7 +49,12 @@ if(CMAKE_GENERATOR STREQUAL "Xcode")
 endif()
 
 function(ivw_define_standard_properties)
-    foreach(target ${ARGN})
+    set(options "QT")
```
Comment (ResearchDaniel) [inline cmake/compileoptions.cmake:52]:
```suggestion
    set(options "HIDE_QT_WARNINGS")
```

File: cmake/globalmacros.cmake [hunk 3]
```diff
@@ -472,7 +472,7 @@ endfunction()
 # This it called from the inviwo module CMakeLists.txt 
 # that is included from ivw_register_modules. 
 function(ivw_create_module)
-    set(options "NO_PCH")
+    set(options "NO_PCH" "QT")
```
Comment (ResearchDaniel) [inline cmake/globalmacros.cmake:475]:
```suggestion
    set(options "NO_PCH" "HIDE_QT_WARNINGS")
```

**Review by ResearchDaniel — DISMISSED** [review]

File: cmake/globalmacros.cmake (hunk 3 above)
Comment (petersteneteg) [reply cmake/globalmacros.cmake:475]: My thinking was that we might use the option for more cases than just the hiding that specific warning.

**Review by petersteneteg — APPROVED** [review]

**Pull request merged**